packages/ledgerstate/branch.go

```diff
@@ -567,24 +532,27 @@ func (c *ChildBranch) Bytes() (marshaledChildBranch []byte) {
 // String returns a human readable version of the ChildBranch.
 func (c *ChildBranch) String() (humanReadableChildBranch string) {
 	return stringify.Struct("ChildBranch",
-		stringify.StructField("parentBranchID", c.ParentBranchID()),
-		stringify.StructField("childBranchID", c.ChildBranchID()),
+		stringify.StructField("ParentBranchID", c.ParentBranchID()),
+		stringify.StructField("ChildBranchID", c.ChildBranchID()),
 	)
 }
 
 // ObjectStorageKey returns the key that is used to store the object in the database. It is required to match the
 // StorableObject interface.
-func (c *ChildBranch) ObjectStorageKey() (objectStorageKey []byte) {
-	return marshalutil.New(BranchIDLength + BranchIDLength).
-		WriteBytes(c.parentBranchID.Bytes()).
-		WriteBytes(c.childBranchID.Bytes()).
-		Bytes()
+func (c *ChildBranch) ObjectStorageKey() []byte {
+	objBytes, err := serix.DefaultAPI.Encode(context.Background(), c, serix.WithValidation())
```

Comment:
I added it everywhere so we don't have to remember which structs have some validation rules and which don't. Could also be confusing in the future when structs will need to be validated and it could be easy to forget to enable validation in serix. But correct - in most places it's unnecessary, but shouldn't cause much overhead.